examples/form_inputs/app.R

```diff
@@ -0,0 +1,147 @@
+library(shiny)
+library(shiny.semantic)
+
+ui <- shinyUI(
+  semanticPage(
+    tags$script(src = "shiny-semantic-radiogroup.js"),
+    suppressDependencies("bootstrap"),
```

Comment:
is this needed here?

Reply:
Nah, that can be removed with the JS file